ci/tasks/azure-provision.yml

```diff
@@ -15,6 +15,5 @@ params:
     AZURE_REGION_SHORT_NAME: replace-me
     AZURE_STORAGE_ACCOUNT_NAME: replace-me
     AZURE_SUBSCRIPTION_ID: replace-me
-    AZURE_TENANT_ID: replace-me
```

Comment:
AZURE_TENANT_ID is already there:
https://github.com/sedouard/bosh-azure-cpi-release/blob/feat_3/ci/tasks/azure-provision.yml#L11